var/spack/repos/builtin/packages/smtk/package.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Smtk(CMakePackage):
+    """SMTK is an open-source, multi-platform Simulation Modeling
+    Toolkit. This package includes libraries and plugins for
+    working with CAD and other types of models used in simulation
+    set up.
+    """
+
+    homepage = "https://www.computationalmodelbuilder.org/"
+    url = "https://gitlab.kitware.com/cmb/smtk"
+    git = "https://gitlab.kitware.com/cmb/smtk.git"
+
+    maintainers = ["kwryankrattiger"]
+    tags = ["cmb"]
+
+    version("master", branch="master", submodules=True)
+
+    releases = [
+        "22.08.0",
```

Comment:
There are additional tags `v23.01.0`, `v23.04.0`, and `v24.01.0` available now. Again, dunno if it matters.